railties/lib/rails/application/configuration.rb

```diff
@@ -236,6 +236,7 @@ def load_defaults(target_version)
               " -frames:v 1 -f image2"
 
             active_storage.variant_processor = :vips
+            active_storage.multiple_file_field_include_hidden = true
```

Comment:
The [Form Builder test coverage](https://github.com/rails/rails/pull/43511/files#diff-f894e0588bffeeff106c8c3a032f7b74a75b4385e14bf5f0da83fec470ece4cb) exercises reading from the value, but I haven't added Railties test coverage to make sure that the configuration values match and the value defaults to what we want. Is that something I should include in this PR as well?